picocli-codegen/src/main/java/picocli/codegen/docgen/manpage/ManPageGenerator.java

```diff
@@ -403,6 +403,12 @@ static void genOptions(PrintWriter pw, CommandSpec spec) {
             writeOption(pw, optionRenderer, paramLabelRenderer, option);
         }
 
+        if (spec.usageMessage().showEndOfOptionsDelimiterInUsageHelp()) {
```

Comment:
We want to show the Options section with the end-of-options delimiter in the man page if the command has no options, but `@Command(showEndOfOptionsDelimiterInUsageHelp = true)`.
 
Higher up in this method, there is this check:

```java
        if (options.isEmpty()) {
            return;
        }
```

We should change that to:

```java
        if (options.isEmpty() && !spec.usageMessage().showEndOfOptionsDelimiterInUsageHelp()) {
            return;
        }
```

It would be great if you can also add a test for this case.

Reply:
Thanks for your guidance, I have submitted the changes. 😀